src/main/java/com/box/sdk/BoxTransactionalAPIConnection.java

```diff
@@ -0,0 +1,166 @@
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope) {
+        return BoxTransactionalAPIConnection.getTransactionConnection(accessToken, scope, null);
+    }
+
+    /**
+     * Request a scoped transactional token for a particular resource.
+     * @param accessToken application access token.
+     * @param scope scope of transactional token.
+     * @param resource resource transactional token has access to.
+     * @return a BoxAPIConnection which can be used to perform transactional requests.
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope, String resource) {
+        BoxAPIConnection apiConnection = new BoxAPIConnection(accessToken);
+
+        URL url;
+        try {
+            url = new URL(apiConnection.getTokenURL());
+        } catch (MalformedURLException e) {
+            assert false : "An invalid token URL indicates a bug in the SDK.";
+            throw new RuntimeException("An invalid token URL indicates a bug in the SDK.", e);
+        }
+
+        String urlParameters;
+        try {
+            urlParameters = String.format("grant_type=%s&subject_token=%s&subject_token_type=%s&scope=%s", GRANT_TYPE,
+                    accessToken, SUBJECT_TOKEN_TYPE, URLEncoder.encode(scope, "UTF-8"));
+
+            if (resource != null) {
+                urlParameters += "&resource=" + URLEncoder.encode(resource, "UTF-8");
+            }
+        } catch (UnsupportedEncodingException e) {
+            throw new BoxAPIException(
+                    "An error occurred while attempting to encode url parameters for a transactional token request"
+            );
+        }
+
+        BoxAPIRequest request = new BoxAPIRequest(apiConnection, url, "POST");
+        request.shouldAuthenticate(false);
+        request.setBody(urlParameters);
+
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+
+        final String fileToken = responseJSON.get("access_token").asString();
+        BoxTransactionalAPIConnection transactionConnection = new BoxTransactionalAPIConnection(fileToken);
+        transactionConnection.setExpires(responseJSON.get("expires_in").asLong() * 1000);
+
+        return transactionConnection;
+    }
+
+    /**
+     * Disabling the non-Box Developer Edition authenticate method.
+     * @param authCode an auth code obtained from the first half of the OAuth process.
+     * @throws UnsupportedOperationException Box Transactional API does not support authentication with an auth code
+     */
+    public void authenticate(String authCode) {
+        throw new UnsupportedOperationException(
+                "BoxTransactionalAPIConnection does not support the authenticate method."
+        );
+    }
+
+    /**
+     * BoxTransactionalAPIConnection can never refresh.
+     * @return true always.
```

Comment:
I think you add @Override annotation to overriden classes. http://docs.oracle.com/javase/6/docs/api/java/lang/Override.html
https://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#@inheritDoc